app/gateway/2.8.x/plan-and-deploy/security/secrets-management/getting-started.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Kong Secrets Management
+---
+
+{:.warning .no-icon}
+> **Beta warning:** 
+> In this version of the feature, you have to explicitly enable this feature. 
+> <br>
+> <br>
+> KONG_VAULTS=vault-name
+> <br>
+> ⚠️This feature is currently in beta state and isn't fully supported. APIs are subject to change.
+
+## Overview
+
+todo: do we want a preface on _why_ secrets management is useful and what it actually is?
+
+Kong offers a mechanism to reference secrets that are stored "somewhere else"(remotely isn't accurate here as "env" and "file" stores them locally).
+
+## Getting Started
+
+In the following example we use the most basic form of secrets management. We are storing our secrets in an "Environment Variable".
+
+A list of supported vault "backend" implementations can be found [here](/gateway/2.8.x/plan-and-deploy/security/secrets-management/backends-overview)
+
+
+In this example we'll replace our plaintext password to our Postgres database with a reference. To do so, please define your environment variable and assign a secret value to it.
+
+
+### Defining a reference
+
+```bash
+export MY_SECRET_POSTGRES_PASSWORD="opensesame"
+```
+
+For Kong to be able to find this secret we have to set up a `reference` to this environment variable. We use a Uniform Resource Locator (URL) format for this. [See the Reference Format Section](/gateway/2.8.x/plan-and-deploy/security/secrets-management/reference-format)
```

Comment:
```suggestion
Next, set up a `reference` to this environment variable so that Kong Gateway can find this secret. We use a Uniform Resource Locator (URL) format for this. 
```